src/cgame/cg_main.cpp

```diff
@@ -1313,17 +1308,26 @@ void CG_Init( int serverMessageNum, int clientNum, const glconfig_t& gl, const G
 
 	CG_SetMapNameFromServerinfo();
 
+	CG_UpdateLoadingStep( LOAD_GEOMETRY );
 	// load the new map
 	CM_LoadMap(cgs.mapname);
 
 	CG_InitMinimap();
 
+	/* Always do this before loading any texture, as texture options may have
```

Comment:
If you are going to say this please link to the issue/PR, since this code is not merged and may end up working in a different way, or have that issue fixed soon afterward.

Reply:
Done.